ui/app/appointments/controllers/manage/appointmentsSummaryController.js

```diff
@@ -3,15 +3,19 @@
 angular.module('bahmni.appointments')
     .controller('AppointmentsSummaryController', ['$scope', '$state', '$window', 'spinner', 'appointmentsService', 'appService',
         function ($scope, $state, $window, spinner, appointmentsService, appService) {
-            var init = function () {
-                $scope.viewDate = $state.params.viewDate || moment().startOf('day').toDate();
-                $scope.weekStartDate = moment($scope.viewDate).startOf('week').toDate();
-                $scope.weekEndDate = moment($scope.viewDate).endOf('week').toDate();
-                $scope.weekStart = appService.getAppDescriptor().getConfigValue('weekStart');
-                $scope.getAppointmentsSummaryForAWeek($scope.weekStartDate, $scope.weekEndDate);
-            };
+            /* var init = function () {
```

Comment:
Same comment here about code commenting. I think better we remove this section of code if it is not going to be used anymore. What do you think?